### PR TITLE
Add additional adducts

### PR DESCRIPTION
**Pull request by jorainer**
@michaelwitting , please have a look if the adduct definition and formula add/sub I added are in the correct/expected format.

### CODE REVIEW

File: inst/adducts/adduct_definition.txt [hunk 1]
```diff
@@ -48,3 +48,7 @@
 "[2M+CHO2]-"	2	44.9982	"CHO2"	"C0"	-1	FALSE
 "[2M+C2H3O2]-"	2	59.01385	"C2H3O2"	"C0"	-1	FALSE
 "[3M-H]-"	3	-1.007276	"C0"	"H"	-1	FALSE
+"[M+H-NH3]+"	1	-16.01927265	"H"	"NH3"	1	TRUE
+"[M-H+HCOOH]-"	1	44.998194	"HCOOH"	"H"	-1	FALSE
```
Comment (michaelwitting) [inline inst/adducts/adduct_definition.txt:52]: This is the same as [M+CHO2]- above. This is a huge problem of non-standardized adduct names

**Review by michaelwitting — COMMENTED** [review]
All other adducts, except for [M-H+HCOOH]- are okay.

File: inst/adducts/adduct_definition.txt (hunk 1 above)
Comment (jorainer) [reply inst/adducts/adduct_definition.txt:52]: Great! Thanks @michaelwitting! That's why I wanted to let you have a look at this. 
So I drop the M-H+HCOOH but keep the M+H-NH3?

**Comment by jorainer** [comment]
I've now also added [M-H+HCOONa]- as we also see that sometimes. Could you please cross-check that it's correct

**Review by michaelwitting — APPROVED** [review]
Looks all good. My main concern are the name of the adducts. There is currently now standardized way to write them, but this might be something we should keep in mind.

**Pull request merged**